src/EnergyPlus/UtilityRoutines.hh

```diff
@@ -573,6 +573,12 @@ namespace UtilityRoutines {
 
     inline bool exists(const std::string& filename);
 
+    bool ValidateFuelType(std::string const &FuelTypeInput, std::string &FuelTypeOutput, bool &FuelTypeErrorsFound);
+
+    bool ValidateFuelTypeWithFuelTypeNum(std::string const &FuelTypeInput, int &FuelTypeNum, bool &FuelTypeErrorsFound);
+
+    bool ValidateFuelTypeWithAssignResourceTypeNum(std::string const &FuelTypeInput, std::string &FuelTypeOutput, int &FuelTypeNum, bool &FuelTypeErrorsFound);
+
```

Comment:
@dareumnam I know this it merged already, but I'm wondering why there are three separate functions here? Seems like these could be collapsed into one function, or at least build off each other so that there's only one place with ifs on the various fuel type strings.

Also, there is still a similar function in [`ExteriorEnergyUse::ValidateFuelType`](https://github.com/NREL/EnergyPlus/blob/8e084f1905da7c4c88baba8b67fcdf3bdd066dab/src/EnergyPlus/ExteriorEnergyUse.cc#L449-L454).

Reply:
@mjwitte Thanks for your comment. This was to remain each object as original as possible, particularly setup output variables and error messages. As you comment, they can be collapsed into a single function if we want to do that, but it needs every object to get fixed. I can do that if it is needed.

Reply:
@dareumnam I would like to finish addressing #7622 before the I/O freeze on Aug 19. The changes for Fuel Oil are waiting for final review in #8134. Then we can build on that and finish up all of the fuel/meter naming work.

Reply:
@mjwitte I see. Then I'll wrap up #7622 right after #8134 gets merged. Thanks for letting me know.

Reply:
@dareumnam Also, note a slight change from the original plan. For output variables we will now use the condensed PascalCase names, not the expanded names. For example "Boiler NaturalGas Energy". This is different from the original plan that was laid out in #7622. The only place expanded names will be used is for table headings (Natural Gas).